code-name-taran/src/com/spaghettisoft/component/hangman/WordToGuess.java

```diff
@@ -0,0 +1,109 @@
+package com.spaghettisoft.component.hangman;
+
+import java.io.IOException;
+import java.io.FileReader;
+import java.io.BufferedReader;
+import java.io.File;
+
+public class WordToGuess {
+
+	private static final String FILE_NAME = "dict.txt.txt";
+	protected static String[] AVAILABLE_WORDS;
+	private String theWord;
+
+	public WordToGuess() throws IOException {
+		AVAILABLE_WORDS = openFromFile();
+		this.theWord = getRandomWord();
+	}
+
+	private String[] openFromFile() throws IOException {
+		File file = new File(FILE_NAME);
+		FileReader fr = new FileReader(file);
+		BufferedReader textReader = new BufferedReader(fr);
+
+		int numberOfLines = readLines();

```

Comment:
Reading how many lines the file has only to create an array is unnecessary in this case. Better to use an ArrayList.